### PR TITLE
Various improvements to minifollowup tables

### PR DESCRIPTION
**Pull request by GarethCabournDavies**
Added a few features to the minifollowup (coincinfo, snglinfo, injinfo) tables:
 - Added chi_eff and chi_p to injinfo
 - Allowing the user to specify a maximum number of columns in the table (as these are starting to get long!), and therefore adding row names
 - Using proper sngl_ranking in the pycbc_page_coincinfo table
 - Silencing some warnings in snglinfo regarding using single-entry arrays instead of floats

## Standard information about the request

This is an enhancement to an existing feature

This change affects the offline search minifollowups.
Some of the functionality is used in inference, but the use-cases there are unaffected

This change changes result presentation

This change follows style guidelines (See e.g. [PEP8](https://peps.python.org/pep-0008/)), has been proposed using the [contribution guidelines](https://github.com/gwastro/pycbc/blob/master/CONTRIBUTING.md)

## Motivation
This is mainly to make things easier when looking at result tables

## Contents
 - Add new entries to the injinfo table 
 - Alter the table template and how it is called so that we can limit the number of columns
 - Add row labels (and remove these from the DQ summary link)

## Links to any issues or associated PRs
#4837 

## Testing performed
Running the codes with appropriate arguments

- [x] The author of this pull request confirms they will adhere to the [code of conduct](https://github.com/gwastro/pycbc/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by spxiwh** [comment]
Example outputs?

**Comment by GarethCabournDavies** [comment]
As discussed, I will try to add the template fits information here

**Review by spxiwh — APPROVED** [review]
Okay, so it looks like the table will wrap and be double the height now (if there are many columns). Does this look okay on a result page? ... Could we maybe have it scroll or do something interactive?

Other than cosmetic concerns I think this is good.

**Comment by GarethCabournDavies** [comment]
I've added scrollable tables; see https://ldas-jobs.ligo.caltech.edu/~gareth.cabourndavies/testoutput/injinfo_extraparams/index.html

I will wait for the CI to run and check through the results pages to see if there are other tables which _shouldnt_ be affected by this (which will be a little more complicated)

**Comment by GarethCabournDavies** [comment]
No other tables affected, but the IFO information is now duplicated if the dq summary link option is not used. Small update, will merge after this is done

**Pull request merged**
